Applications/FileManager/FileUtils.cpp

```diff
@@ -86,6 +87,8 @@ bool copy_file_or_directory(const String& src_path, const String& dst_path)
         return false;
     }
 
+    ScopeGuard close_fd_guard([src_fd]() { close(src_fd); });
```

Comment:
This is awkward because it will also be closed in `copy_file()` (but not in `copy_directory()`)
A double `close()` is not the end of the world here of course.

Reply:
A double close is the end of the world, especially in a multithreaded program like the FileManager.

Reply:
I'm having some serious race condition blindness lately. Thanks for setting me straight :)